platforms/osx/README.md

```diff
@@ -0,0 +1,59 @@
+Mac OS X
+========
+
+## Setup ##
+
+This project uses CMake (minimum version 3.0), you can download it [here](http://www.cmake.org/download/) or use your favorite installation package tool like [homebrew](http://brew.sh/).
+
+```bash
+brew install cmake
+```
+
+Make sure to update git submodules before you build:
+
+```bash
+git submodule update --init
+```
+
+## Build ##
+
+There are several ways you can build the tangram-es library and demo application on Mac OS X:
+
+### Command Line ###
+
+To build a runnable OS X application bundle, run:
+
+```bash
+make osx
+```
+And open the application with:
+
+```bash
+open build/osx/bin/tangram.app
```

Comment:
Can we also mention about the use of `-f` command line argument to specify a scene file (remote)?